### ISSUE TITLE
What is Synthetic Self-Training?

### ISSUE THREAD

**Issue by hsm207** [huggingface/pytorch-pretrained-BERT#370]
The current best performing model on[ SQuAD 2.0](https://rajpurkar.github.io/SQuAD-explorer/) is BERT + N-Gram Masking + Synthetic Self-Training (ensemble):

![image](https://user-images.githubusercontent.com/2398765/54234467-24466380-454a-11e9-8674-d9e7004da027.png)

What is Synthetic Self-Training?

**Comment by thomwolf** [comment]
Check Jacob Devlin slides starting from slide 26 [here](https://nlp.stanford.edu/seminar/details/jdevlin.pdf?fbclid=IwAR2TBFCJOeZ9cGhxB-z5cJJ17vHN4W25oWsjI8NqJoTEmlYIYEKG7oh4tlY)

**Comment by hsm207** [comment]
@thomwolf thanks, the slides were helpful. Do you know if there is a recording of the talk publicly available somewhere?

**Comment by thomwolf** [comment]
I don't know!

**Comment by CatalinVoss** [comment]
It was a very crowded room and these talks are generally not recorded, sorry…

**Comment by Dogy06** [comment]
Is the synthetic self training module in this code?

**Comment by jind11** [comment]
Same question, will the synthetic self training module be in this code?

**Comment by thomwolf** [comment]
Presently not in the library and there are no short-term plans to add synthetic self-training to the library.
